src/screens/ExchangeModal.js

```diff
@@ -291,8 +291,9 @@ export default function ExchangeModal({
   useEffect(() => {
     if (!isEmpty(gasFeeParamsBySpeed)) {
       updateGasLimit();
+      updateTxFee();
     }
-  }, [gasFeeParamsBySpeed, updateGasLimit]);
+  }, [gasFeeParamsBySpeed, updateGasLimit, updateTxFee]);
```

Comment:
were you seeing an error? `updateTxFee` is called inside of `updateGasLimit`

Reply:
taking this stuff out to take a look with the other gas stuff i need to fix